docs/configuration-reference.md

```diff
@@ -17,6 +17,8 @@ The SR-IOV CNI configures networks through a CNI spec configuration object. In a
 * `min_tx_rate` (int, optional): change the allowed minimum transmit bandwidth, in Mbps, for the VF. Setting this to 0 disables rate limiting. The min_tx_rate value should be <= max_tx_rate. Support of this feature depends on NICs and drivers.
 * `max_tx_rate` (int, optional): change the allowed maximum transmit bandwidth, in Mbps, for the VF.
 Setting this to 0 disables rate limiting.
+* `loglevel` (string, optional): either of panic, error, warning, info, debug with a default of error.
```

Comment:
Because I originally did not want to print any extra information by default with the introduction of the logging mechanism. That's also the reason why I set logging.Info("Cannot load config file from cache"). Let me switch it to Info